chart/cas-cif/templates/secret.yaml

```diff
@@ -10,6 +13,15 @@
 {{- $dbOwnerPassword = index $secret.data "database-owner-password" }} {{/* # pragma: allowlist secret */}}
 {{- end -}}
 
+{{- $ggircsSecret := (lookup "v1" "Secret" "9212c9-dev" "cas-ggircs" ) }}
+{{- if $ggircsSecret }}
+{{- $dbReadonlyUser = index $ggircsSecret.data "database-readonly-user" }} {{/* # pragma: allowlist secret */}}
```

Comment:
I believe all of these #pragma directives can be removed now that we're not using detect-secrets anymore